### ISSUE TITLE
Add support for Anthropic Claude 3.5 Sonnet

### ISSUE THREAD

**Issue by WPrintz** [go-micah/chat-cli#47]
Would be good if chat-cli allowed use of Claude 3.5 in the model choice optionset.

**Comment by micahwalter** [comment]
Yes, I have been meaning to update to add support for a number of new models. Will start with this one soon.

**Comment by micahwalter** [comment]
Fixed in #48
